packages/desktop-client/src/components/manager/subscribe/Login.tsx

```diff
@@ -1,42 +1,210 @@
 // @ts-strict-ignore
 import React, { useState, useEffect } from 'react';
 import { useDispatch } from 'react-redux';
-import { useParams, useSearchParams } from 'react-router-dom';
+import { useSearchParams } from 'react-router-dom';
 
-import { createBudget } from 'loot-core/src/client/actions/budgets';
 import { loggedIn } from 'loot-core/src/client/actions/user';
 import { send } from 'loot-core/src/platform/client/fetch';
+import { type OpenIdConfig } from 'loot-core/types/models/openid';
 
+import { useNavigate } from '../../../hooks/useNavigate';
 import { AnimatedLoading } from '../../../icons/AnimatedLoading';
-import { theme } from '../../../style';
+import { styles, theme } from '../../../style';
 import { Button, ButtonWithLoading } from '../../common/Button2';
 import { BigInput } from '../../common/Input';
+import { Label } from '../../common/Label';
 import { Link } from '../../common/Link';
+import { Select } from '../../common/Select';
 import { Text } from '../../common/Text';
 import { View } from '../../common/View';
+import { useAvailableLoginMethods, useLoginMethod } from '../../ServerContext';
 
 import { useBootstrapped, Title } from './common';
+import { OpenIdForm } from './OpenIdForm';
+
+function PasswordLogin({ setError, dispatch }) {
+  const [password, setPassword] = useState('');
+  const [loading, setLoading] = useState(false);
+
+  async function onSubmitPassword() {
+    if (password === '' || loading) {
+      return;
+    }
+
+    setError(null);
+    setLoading(true);
+    const { error } = await send('subscribe-sign-in', {
+      password,
+      loginMethod: 'password',
+    });
+    setLoading(false);
+
+    if (error) {
+      setError(error);
+    } else {
+      dispatch(loggedIn());
+    }
+  }
+
+  return (
+    <View style={{ flexDirection: 'row', marginTop: 5 }}>
+      <BigInput
+        autoFocus={true}
+        placeholder="Password"
+        type="password"
+        onChangeValue={newValue => setPassword(newValue)}
```

Comment:
This is missing:
```
            onEnter={onSubmitPassword}
```

It'll allow us to sign in by pressing enter after entering the password